sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CodeGenerationSuite.scala

```diff
@@ -124,6 +124,9 @@ class CodeGenerationSuite extends SparkFunSuite with ExpressionEvalHelper {
     }
   }
 
+  /**
+  *  [[CreateStruct]] is now a constructor of [[CreateNamedStruct]]
```

Comment:
Lets remove this comment. If people want to know what happened to `CreateStruct` they should check the commit log.